src/main/java/com/prgrms/amabnb/review/repository/QueryReviewRepository.java

```diff
@@ -2,11 +2,18 @@
 
 import java.util.List;
 
+import org.springframework.data.domain.PageRequest;
 import org.springframework.data.domain.Pageable;
 
-import com.prgrms.amabnb.review.dto.request.SearchReviewRequest;
-import com.prgrms.amabnb.review.dto.response.SearchReviewResponse;
+import com.prgrms.amabnb.review.dto.request.SearchMyReviewRequest;
+import com.prgrms.amabnb.review.dto.request.SearchRoomReviewRequest;
+import com.prgrms.amabnb.review.dto.response.SearchMyReviewResponse;
+import com.prgrms.amabnb.review.dto.response.SearchRoomReviewResponse;
 
 public interface QueryReviewRepository {
-    List<SearchReviewResponse> findAllByCondition(Long userId, SearchReviewRequest condition, Pageable pageable);
+    List<SearchMyReviewResponse> findMyReviewByCondition(
+        Long userId, SearchMyReviewRequest condition, Pageable pageable);
+
+    List<SearchRoomReviewResponse> findRoomReviewByCondition(
+        Long roomId, SearchRoomReviewRequest condition, PageRequest pageable);
```

Comment:
```suggestion
        Long roomId, SearchRoomReviewRequest condition, Pageable pageable);
```
Pageable로 파라미터를 받는 게 어떨까요?

Reply:
감사합니당!